tests.js

```diff
@@ -54,6 +54,955 @@ window.Specs = {
+				"links": {
+					"tr": "#propdef-border-spacing",
+					"dev": "#propdef-border-spacing"
+				},
+				"tests": ["10px", "1em 0.5cm"]
+			},
+			"caption-side": {
+				"links": {
+					"tr": "#caption-position",
+					"dev": "#caption-position"
+				},
+				"tests": ["top", "bottom"]
+			},
+			"empty-cells": {
+				"links": {
+					"tr": "#empty-cells",
+					"dev": "#empty-cells"
+				},
+				"tests": ["show", "hide"]
+			},
+			"table-layout": {
+				"links": {
+					"tr": "#width-layout",
+					"dev": "#width-layout"
+				},
+				"tests": ["auto", "fixed"]
+			}
+		}
+	},
+
+	"css2-text": {
+		"title": "CSS 2 Text",
+		"links": {
+			"tr": "CSS22/text.html",
+			"dev": "css2/"
+		},
+		"status": {
+			"stability": "stable",
+			"first-snapshot": 1998,
+			"last-snapshot": 1998
+		},
+		"properties": {
+			"letter-spacing": {
+				"links": {
+					"tr": "#propdef-letter-spacing",
+					"dev": "#propdef-letter-spacing"
+				},
+				"tests": ["normal", "10px"]
+			},
+			"text-align": {
+				"links": {
+					"tr": "#alignment-prop",
+					"dev": "#alignment-prop"
+				},
+				"tests": ["left", "right", "center", "justify"]
+			},
+			"text-decoration": {
+				"links": {
+					"tr": "#lining-striking-props",
+					"dev": "#lining-striking-props"
+				},
+				"tests": [
+					"none", "underline", "overline", "line-through", "blink",
+					"underline overline", "underline overline line-through"
+				]
+			},
+			"text-indent": {
+				"links": {
+					"tr": "#indentation-prop",
+					"dev": "#indentation-prop"
+				},
+				"tests": ["10px", "10%"]
+			},
+			"text-transform": {
+				"links": {
+					"tr": "#caps-prop",
+					"dev": "#caps-prop"
+				},
+				"tests": ["none", "capitalize", "uppercase", "lowercase"]
+			},
+			"white-space": {
+				"links": {
+					"tr": "#white-space-prop",
+					"dev": "#white-space-prop"
+				},
+				"tests": ["normal", "pre", "nowrap", "pre-wrap", "pre-line"]
+			},
+			"word-spacing": {
+				"links": {
+					"tr": "#propdef-word-spacing",
+					"dev": "#propdef-word-spacing"
+				},
+				"tests": ["normal", "10px"]
+			}
+		}
+	},
+
+	"css2-ui": {
+		"title": "CSS 2 User Interface",
+		"links": {
+			"tr": "CSS22/ui.html",
+			"dev": "css2/"
+		},
+		"status": {
+			"stability": "stable",
+			"first-snapshot": 1998,
+			"last-snapshot": 1998
+		},
+		"properties": {
+			"cursor": {
+				"links": {
+					"tr": "#cursor-props",
+					"dev": "#cursor-props"
+				},
+				"tests": [
+					"auto", "crosshair", "default", "pointer", "move", "e-resize",
+					"ne-resize", "nw-resize", "n-resize", "se-resize", "sw-resize",
+					"s-resize", "w-resize", "text", "wait", "help", "progress",
+					"url(cursor.png), auto", "url(cursor.svg), url(cursor.png), pointer"
+				]
+			},
+			"outline-color": {
+				"links": {
+					"tr": "#dynamic-outlines",
+					"dev": "#dynamic-outlines"
+				},
+				"tests": [
+					"black", "#00f", "#000000", "rgb(255, 255, 255)",
+					"rgb(100%, 50%, 50%)", "transparent"
```

Comment:
`transparent` was also not valid here yet. But the keyword `invert` should be tested.

Reply:
Correct! I've replaced `transparent` by `invert`.